composer.json

```diff
@@ -93,7 +93,7 @@
         "google/cloud-storage": "^1.10",
         "guzzlehttp/guzzle": "^6.3",
         "helios-ag/fm-elfinder-bundle": "^10.0.4",
-        "heureka/overeno-zakazniky": "^2.0.6",
+        "heureka/overeno-zakazniky": "^4.0.1",
```

Comment:
This needs to be done also in project-base and framework. Because you are updating project-base, you need to also add upgrade notes.

Reply:
You still forgot to add it to framework